src/main/java/com/techconative/restel/utils/RestelUtils.java

```diff
@@ -238,6 +254,21 @@ private static void validate(TestApiDefinitions testApiDefinitions) {
     }
   }
 
+  private static void validate(TestApiWrappers testApiWrappers) {
+    if (StringUtils.isEmpty(testApiWrappers.getTestApiName())) {
+      throw new RestelException("TEST_API_NAME_EMPTY");
+    }
+    if (StringUtils.isEmpty(testApiWrappers.getTestApiWrapperName())) {
+      throw new RestelException("TEST_API_WRAPPER_NAME_EMPTY");
+    }
+    if (StringUtils.isEmpty(testApiWrappers.getTestApiWrapperDescription())) {
+      throw new RestelException("TEST_API_WRAPPER_DESC_EMPTY");
+    }
+    if (StringUtils.isEmpty(testApiWrappers.getTestApiWrapperParameters())) {
+      throw new RestelException("TEST_API_WRAPPER_PARAM_EMPTY");
+    }
```

Comment:
Can the if-ladder be avoided? Probably we can have a `Map<Predicate<TestApiWrappers>,String>` containing the check to missing message and iterate through it to check everything here.

I understand this is how it has been done in other places. But in general it's a good practice to separate _how_
with _what_ to the extent possible.